cpp/proto/serializer.cc

```diff
@@ -316,35 +251,36 @@ SerializeResult TLSSerializer::WriteSCTV2(
   if (sct.id().key_id().size() != Serializer::kKeyIDLengthInBytes) {
     return SerializeResult::INVALID_KEYID_LENGTH;
   }
-  WriteUint(sct.version(), Serializer::kVersionLengthInBytes);
-  WriteFixedBytes(sct.id().key_id());
-  WriteUint(sct.timestamp(), Serializer::kTimestampLengthInBytes);
+  WriteUint(sct.version(), Serializer::kVersionLengthInBytes,
```

Comment:
This one also should NOT clear `output`, right? Ugh, I hate subtle changes like that...